scripts/build-win.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs')
+
+const dir = './apks'
+
+if (!fs.existsSync(dir)){
+	fs.mkdirSync(dir)
+}else{
```

Comment:
Please format the file properly

Reply:
I made more changes to the file, if the format is still not good , please let me know